crypt.go

```diff
@@ -641,13 +693,13 @@ type InitializeLUKS2ContainerOptions struct {
 	// expressed in multiples of 1024 bytes. The value must be aligned to
 	// 4096 bytes, with the maximum size of 128MB.
 	KeyslotsAreaKiBSize int
+
+	// KDFOptions sets the KDF options for the initial keyslot. If this
+	// is nil then the defaults are used.
+	KDFOptions *KDFOptions
```

Comment:
in master this struct remains the same, and the KDFoptions are passed in separately.